cmd/infra/powervs/destroy.go

```diff
@@ -416,11 +419,14 @@ func destroyPowerVsCloudInstance(ctx context.Context, options *DestroyInfraOptio
 
 // monitorPowerVsJob monitoring the submitted deletion job
 func monitorPowerVsJob(id string, client *instance.IBMPIJobClient, infraID string, timeout time.Duration) error {
-
 	f := func() (bool, error) {
 		job, err := client.Get(id)
 		if err != nil {
-			return false, err
+			if err := isNotRetryableError(err, timeoutErrorKeywords); err != nil {
```

Comment:
here I see you are declaring the local `err` but not in other place, can this be consistent everywhere?

Reply:
Modified, Ptal now!